src/queries.ts

```diff
@@ -149,7 +146,8 @@ function createQuery(minBlock: number, maxBlock: number): Query {
 							values: [${VALID_APP_NAMES.map((appName) => `"${appName}"`)}]
```

Comment:
Were we going to query per app or continue querying all app names in one go? I recall some gateway DB issues with queries like this, but they may be patched up right now